http_service/listener.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Generate your credentials in https://pulseguardian.mozilla.org
+
+Call this script like:
+  export PULSE_USER=generated_username
+  export PULSE_PASSWORD=generated_username
+  # In case you want to hit the live server
+  export BUGBUG_HTTP_SERVER=https://bugbug.herokuapp.com
+  cd http_service && docker-compose run bugbug-http-service
+"""
+import logging
+import os
+import traceback
+
+import requests
+from pulse.consumers import ConsumerFactory
+
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+BUGBUG_HTTP_SERVER = os.environ.get("BUGBUG_HTTP_SERVER", "http://localhost:8000")
+
+
+def _on_message(body, message):
+    try:
+        branch = body["payload"]["data"]["repo_url"].split("/")[-1]
+        rev = body["payload"]["data"]["heads"][0]
+        if branch in ["autoland", "try"]:
+            url = "{}/push/{}/{}/schedules".format(BUGBUG_HTTP_SERVER, branch, rev)
+            response = requests.get(url, headers={"X-Api-Key": "pulse_listener"})
```

Comment:
I was thinking we'd directly put the job in the queue, but this should work exactly as well.